src/components/boardPage/TaskCard.tsx/TaskCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./TaskCard.scss";
+import { ITask, TaskStatusEnum } from "../../../interfaces/task";
+
+interface TaskCard extends ITask {}
+
+export const TaskCard: React.FC<ITask> = ({
+  assignedQuantity,
+  creatorName,
+  status,
+  taskId,
+  title,
+}) => {
+  const statusComponent = () => {
+    if (status === TaskStatusEnum.complete) {
+      return (
+        <div className="bg-success rounded px-2 py-1 text-white">
+          <p className="p-0 m-0">
+            <small>Completed</small>
+          </p>
+        </div>
+      );
+    } else {
+      return (
+        <div className="bg-secondary rounded px-2 py-1 text-white">
+          <p className="p-0 m-0">
+            <small>Incomplete</small>
+          </p>
+        </div>
+      );
+    }
+  };
+
+  return (
+    <div className="rounded border px-2 py-3 mb-3 bg-light">
+      <small className="text-muted fst-italic">Task</small>
+      <p>{title}</p>
+      <hr />
+      <p>
+        <strong>Author: </strong>
+        {creatorName}
+      </p>
+
+      <div className="d-flex justify-content-between">
+        <p className="p-0 m-0">{assignedQuantity} people</p>
+        {statusComponent()}
```

Comment:
Think if you would like to be able to include translations. There are several tools you could include. If you decide to refrain from that, it is another option.

Reply:
https://lokalise.com/blog/react-i18n-intl/

Reply:
The "statusComponent" is not really a component, it is a function, I did it like that to reduce code on the JSX area since on that function I have a conditional.